src/app/components/play-round/play-round.component.scss

```
@@ -60,14 +60,17 @@
   }
 }
 
-.marker-name {
+
+.marker-info {
   margin-top: 16px;
   margin-bottom: 0;
   margin-right: 8px;
   margin-left: 8px;
+}
+
+.marker-name, .marker-tip {
   text-align: center;
 
-  @include responsive-font-size(2rem, 2.5rem, 3rem);
   font-weight: bold;
```

Sorry for this nitpicking but I think we should use a lower font-weight. bold (which is 700) looks like a bit much, especially on smaller screens and with the smaller tip. Using this font weight also requires  the browser to get another font file.

```suggestion
  font-weight: 500;
```

---

Sure. I don't have a preference towards either.